test/helpers/kubectl.go

```diff
@@ -899,12 +902,12 @@ func (kub *Kubectl) DumpCiliumCommandOutput(namespace string) {
 // directory
 func (kub *Kubectl) GatherLogs() {
 	reportCmds := map[string]string{
-		"kubectl get pods -o wide --all-namespaces":     "pods.txt",
-		"kubectl get services -o wide --all-namespaces": "svc.txt",
-		"kubectl get ds -o wide --all-namespaces":       "ds.txt",
-		"kubectl get cnp --all-namespaces":              "cnp.txt",
-		"kubectl describe pods --all-namespaces":        "pods_status.txt",
-		"kubectl -n kube-system logs -l k8s-app=cilium": "cilium_logs.txt",
+		"kubectl get pods -o wide --all-namespaces":                  "pods.txt",
+		"kubectl get services -o wide --all-namespaces":              "svc.txt",
+		"kubectl get ds -o wide --all-namespaces":                    "ds.txt",
+		"kubectl get cnp --all-namespaces":                           "cnp.txt",
+		"kubectl describe pods --all-namespaces":                     "pods_status.txt",
+		"kubectl -n kube-system logs -l k8s-app=cilium --timestamps": "cilium_logs.txt",
```

Comment:
6961a5daedeb49ce738e8d42006a7e68a0818eb2 removed the usage of timestamps - might want to remove this for now and then file a GH issue to explore whether we can use `--timestamps` in the future.

Reply:
@ianvernon I think you linked the wrong commit? We're using timestamps elsewhere, there was temporarily a problem when `--timestamps` was specified before the command (ie, `logs`), but after we shuffled that around I don't think we've seen problems, have we?